src/util/fs_helpers.h

```diff
@@ -62,6 +63,9 @@ void ReleaseDirectoryLocks();
 bool TryCreateDirectories(const fs::path& p);
 fs::path GetDefaultDataDir();
 
+std::string PermsToString(fs::perms p);
+std::optional<fs::perms> StringToPerms(const std::string& s);
```

Comment:
In 38af55e285336cdd3f42635938f24622451703b0 "util: add PermsToString and StringToPerms"

nit: This naming is a little bit confusing to me since it suggests that these functions are inverses of each other, but they're not as their string outputs are not compatible.

Reply:
I agree on reflection, gave them new names in 49d5bfdd7e7